db/R/get.trait.data.R

```diff
@@ -97,15 +102,15 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
           if (!file.exists(file.path(files$file_path[[id]], files$file_name[[id]]))) {
             foundallfiles <- FALSE
             logger.error("can not find posterior file: ", file.path(files$file_path[[id]], files$file_name[[id]]))
-          } else if (forceupdate && (files$file_name[[id]] == "species.csv")) {
+          } else if (files$file_name[[id]] == "species.csv") {
```

Comment:
I'm not following the logic of either the original or the revised. If we're doing AUTO, don't we want to do this check? If we're doing a force update, wouldn't we delete the file regardless of the check?  Maybe I'm just missing something. Whomever wrote this originally should take a close look. @dlebauer ?